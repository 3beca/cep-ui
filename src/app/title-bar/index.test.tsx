import React from 'react';
import { render } from '@testing-library/react';
import AppBar from './index';

test('renders TitleBar withou crash', () => {
    const { getByText } = render(<AppBar />);
    const linkElement = getByText(/CEP Service/i);
    expect(linkElement).toBeInTheDocument();
});