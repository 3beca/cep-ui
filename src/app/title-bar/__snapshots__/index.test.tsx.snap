// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleBar snapshot 1`] = `
<div>
  <header
    class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary makeStyles-root-85 mui-fixed MuiPaper-elevation4"
  >
    <div
      class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root makeStyles-icon-86 MuiSvgIcon-fontSizeLarge"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M22 5.72l-4.6-3.86-1.29 1.53 4.6 3.86L22 5.72zM7.88 3.39L6.6 1.86 2 5.71l1.29 1.53 4.59-3.85zM12.5 8H11v6l4.75 2.85.75-1.23-4-2.37V8zM12 4c-4.97 0-9 4.03-9 9s4.02 9 9 9c4.97 0 9-4.03 9-9s-4.03-9-9-9zm0 16c-3.87 0-7-3.13-7-7s3.13-7 7-7 7 3.13 7 7-3.13 7-7 7z"
        />
      </svg>
      <h6
        class="MuiTypography-root MuiTypography-h6 MuiTypography-colorInherit MuiTypography-noWrap"
      >
        CEP Service
      </h6>
    </div>
  </header>
</div>
`;
